README.md

```diff
@@ -18,10 +18,12 @@ Can be installed via `pip`: `pip install codeflare-sdk`
 ### Prerequisites
 
 We recommend using Python 3.9 for development.
-Install development specific dependencies:
-  `$ pip install -r requirements-dev.txt`
+Install dependencies: `poetry install` 
```

Comment:
Should we include instructions for people who want to install just the base dependancies in the pyproject, vs just the test dependencies vs the docs dependancy?

Reply:
Yes we can do this. Currently the docs dependencies are optional so we can use `--with docs` to include them. Test dependencies are not currently optional so will be installed with the base dependencies. Do we want to make them optional? Or note to edit the file to make them optional if not required?

Reply:
Yeah I think that having the test dependencies also be optional, but with a note on how to install those as well, might be the way to go. Because at the base, people just using the package from source should get just the required dependencies to get running by default. And then for developers, they should include the test dependencies themselves, run the `pre-commit install`, etc. As long as we are clear in our README about how to install test/doc dependencies, it should be fine

Reply:
Great, I will update this then. Thank you!